code/controllers/Processes/inactivity.dm

```diff
@@ -1,16 +0,0 @@
-/*/datum/controller/process/inactivity/setup()
```

Comment:
Deleted this too; we've not used this in eons, and it was problematic when it *did* exist.